packages/accounts-password/package.js

```diff
@@ -5,11 +5,12 @@ Package.describe({
   // 2.2.x in the future. The version was also bumped to 2.0.0 temporarily
   // during the Meteor 1.5.1 release process, so versions 2.0.0-beta.2
   // through -beta.5 and -rc.0 have already been published.
-  version: "3.0.3",

```

Comment:
This will be a major version bump.

Reply:
I guess the [CONTRIBUTING.md](https://github.com/meteor/meteor/blob/devel/CONTRIBUTING.md#submitting-pull-requests) should be clarified to handle such cases: here the changes fall in the first category : 
> If your changes are ok to be released without a whole new Meteor version bump just the patch, for example, 2.4.5 will become 2.4.6.

I would have pushed a major update in the version without reading this line ;)

Reply:
Technically you are correct. But in this case it is a major change as it changes a critical piece of the package related to security. Major version bump is always a headache, which in this case we want so that people notice, be aware of this and take action with their users if needed.